File: SimuladoAtividade1/Usuario.java
//Carolina Perez RA:20.00968-2

package SimuladoAtividade1;

public class Usuario {
    private String nome;
    private String email;
    private String senha;

    public Usuario(String nome, String senha, String email){
        this.nome = nome;
        this.senha = senha;
        this.email = email;
    }

    public void visualizarUsuario(){
        System.out.println("Dados do Usuario:");
        System.out.println("Nome:" + nome);
        System.out.println("E-mail:" + email);
        System.out.println("Senha:" + senha);
        
    }

    public String getNome(){
        return nome;
    }

    public void setNome(String nome){
        this.nome = nome;
    }

    public String getsenha(){
        return senha;
    }
    public void setsenha(String senha){
        this.senha = senha;
    }
    public String getEmail(){
        return email;
    }
    public void setEmail(String email){
        this.email = email;
    }
    

}

File: Atividade3/src/models/usuario.py
class Usuario():
    def __init__(self, nome, email, nomeUsuario, senha):
        self._nome = nome
        self._email = email 
        self._nomeUsuario = nomeUsuario
        self._senha = senha

    def get_nome(self):
        return self._nome
    def get_email(self):
        return self._email
    def get_nomeUsuario(self):
        return self._nomeUsuario
    def get_senha(self):
        return self._senha

    def __str__(self) -> str:
        return f'Usuario(Nome: {self._nome}, Email: {self._email}, Senha: {self._senha})'
